src/mixins/dpValidateMixin.js

```diff
@@ -42,7 +42,7 @@ export default {
         const customErrors = invalidFields
           .filter(element => element.hasAttribute('data-dp-validate-error'))
           .map(element => element.dataset.dpValidateError)
-        customErrors.forEach(error => dplan.notify.notify('error', Translator.trans(error)))
+        customErrors.forEach(error => dplan.notify.notify('error', error))
```

Comment:
Is it safe to remove this?

Reply:
I first thought that in core we use `data-dp-validate-error` attribute with `Translator.trans()`, but not in all places!
I make Revert for now!
https://github.com/demos-europe/demosplan-ui/pull/918/commits/33a7c96c0d5c714679ce99cf220201561392dfd3

Reply:
But isn't the goal of this PR to remove all usages of `Translator.trans()`?
I think a better solution would be to add a PR in demosplan-core that adds the Translator in all instances where we use `data-dp-validate-error`.

Reply:
Ok, I will make also that! thanks!

Reply:
Core PR: https://github.com/demos-europe/demosplan-core/pull/3467

Reply:
https://github.com/demos-europe/demosplan-ui/pull/918/commits/662697003e04d568236fde91ed8bb2dd2bb16a7a

Reply:
@hwiem Now you can review this PR again with https://github.com/demos-europe/demosplan-core/pull/3467!

I also deleted Translator.trans from demosplan-ui in DpApi.js and added it with https://github.com/demos-europe/demosplan-core/pull/3467!

Thanks!